master/internal/rm/kubernetesrm/request_workers.go

```diff
@@ -12,61 +13,72 @@ import (
 type requestProcessingWorker struct {
 	podInterfaces       map[string]typedV1.PodInterface
 	configMapInterfaces map[string]typedV1.ConfigMapInterface
+	syslog              *logrus.Entry
 }
 
-func (r *requestProcessingWorker) Receive(ctx *actor.Context) error {
-	switch msg := ctx.Message().(type) {
-	case actor.PreStart:
-		ctx.Tell(ctx.Self().Parent(), workerAvailable{})
-	case actor.PostStop:
-		// This should not happen since the request worker actors would not stop during
-		// the master is running.
+type readyCallbackFunc func(string)
```

Comment:
for clarity, could be nice to type the string and have something like `prev workID`